EasyLM/models/llama/llama_train_ppo.py

```diff
@@ -80,8 +83,14 @@
     # debugging flags
     max_steps_per_epoch=0,
     generate_only=False,
+    no_backward=False,
 )
 
+pad_token_id = 0
+bos_token_id = 1
+eos_token_id = 2
+
+
```

Comment:
Keeping those hard-coded here. There's additional complication with tokenizers and let's review this later.

Reply:
With llama 3 (new bos/eos id), this might cause issues. happy to merge now but we should fix this in the near future.